packages/flutter/test/material/scrollbar_test.dart

```diff
@@ -19,34 +19,49 @@ class TestCanvas implements Canvas {
   }
```

Comment:
Add a general test for the Material Scrollbar as well, to make sure the Scrollbar catches the MediaQuery padding correctly, not just the custom painter.